cyberark/conjur
# Design updates

Closes [relevant GitHub issues, eg #76]

#### What does this pull request do?
Adds content and design updates from initial feedback to Installation flow.
#### Where should the reviewer start?
homepage
#### How should this be manually tested?
- Verify new content on homepage in CTA boxes, 
- Check "contribution" link in sidebar & footer link to GitHub
- Verify 'Built for Containers' content is on homepage
- CyberArl logo added to footer
- Slack footer CTA is properly styled
- Smaller CYBR logo in top navigation

#### Screenshots (if appropriate)
![conjur_org_home](https://user-images.githubusercontent.com/123787/29684721-fa4593b6-88e0-11e7-8c30-ee2fc1bec7dc.png)

#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/design-updates/
#### Questions:
> Does this have automated Cucumber tests?
no

> Can we make a blog post, video, or animated GIF out of this?
no

> Is this explained in documentation?
no

> Does the knowledge base need an update?
no
